medusa/failed_processor.py

```diff
@@ -62,7 +62,7 @@ def process(self):
 
         if segment:
             self.log(logger.DEBUG, u'Adding this release to failed queue: {release}'.format(release=releaseName))
-            cur_failed_queue_item = FailedQueueItem(parsed_result.series, segment)
+            cur_failed_queue_item = FailedQueueItem(parse_result.series, segment)
```

Comment:
was about to commit that. thanks. pycharm was updating

Reply:
No problem.